.github/workflows/docker.yml

```diff
@@ -23,6 +26,7 @@ jobs:
         uses: docker/build-push-action@v2
         with:
           context: .
+          platforms: linux/amd64
```

Comment:
Nit: Remove this line so that we build and test for the runner's platform (which could theoretically change from amd64 to arm64 or something else in the future).

```suggestion
```